app/serializers/pull_request_serializer.rb

```diff
@@ -1,6 +1,6 @@
 class PullRequestSerializer < ActiveModel::Serializer
   attributes :id, :title, :repository_name, :owner_id, :html_url, :likes, :created_at,
-             :owner_name, :description
+             :owner_name, :description, :commits, :owner_login
   def likes
     { total: object.likes.count }
   end
```

Comment:
si solamente tiene el total, porque no mejor retornar solamente ese número? Después desde Vue se podría llamar como `likes` no más y no `likes.total`
```suggestion
  def likes
    object.likes.count
  end
```